ietf/doc/utils_search.py

```diff
@@ -144,7 +144,17 @@ def fill_in_document_table_attributes(docs, have_telechat_date=False):
         l.append(rel_rfc_aliases[rel.source_id])
         l.sort()
 
+def augment_docs_with_related_docs_info(docs):
+    """Augment all documents with related documents information.
+    At first, it handles only conflict review document page count to mirror the original document page count."""
 
+    for d in docs:
+        if d.type_id == 'conflrev':
+            if len(d.related_that_doc('conflrev')) != 1:
```

Comment:
Right now, the other parts of the datatracker that set up conflict reviews will not let a conflict review be created that has anything other than exactly one reviewed document, so this line is a safety check against misconfiguration or changes to that assumption in the future. That's good.

The code could instead have looped over the reviewed documents and summed their pages, but that just makes another assumption about what having multiple related documents might mean might mean in the future.